test/libsolidity/smtCheckerTests/try_catch/try_multiple_returned_values.sol

```diff
@@ -0,0 +1,27 @@
+pragma experimental SMTChecker;
+
+abstract contract D {
+	function d() external virtual returns (uint x, bool b);
+}
+
+contract C {
+
+	int x;
+	D d;
+
+	function f() public {
+		x = 0;
+		try d.d() returns (uint x, bool c) {
+			assert(x == 0); // should fail, x is the local variable shadowing the state variable
```

Comment:
Oh wow, didn't know that this shadowing was allowed

Reply:
The compiler warns about it, but it is allowed.